recipes/embree/all/conanfile.py

```diff
@@ -0,0 +1,101 @@
+
+import os
+from conans import CMake, tools
+from conans.model.conan_file import ConanFile
+
+
+class Embree(ConanFile):
+    name = "embree"
+    version = "3.6.1"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "embree", "raytracing", "rendering")
+    description = "Intel's collection of high-performance ray tracing kernels."
+    generators = "cmake"
+    homepage = "https://embree.github.io/"
+    _source_folder = "source_subfolder"
+    _build_folder = "build_subfolder"
+    exports_sources = "CMakeLists.txt"
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "tbb/2019_u9"
+
+    options = {
+        "fPIC": [
```

Comment:
```suggestion
        "shared": [
        	True,
        	False
        ],
        "fPIC": [
```

Reply:
in addition to fPIC, I added an additional option named shared